ci-operator/step-registry/ipi/conf/vsphere/lb/external/ipi-conf-vsphere-lb-external-commands.sh

```diff
@@ -165,10 +169,17 @@ curl -sSL "https://mirror2.openshift.com/pub/openshift-v4/clients/butane/latest/
 
 LB_VMNAME="${cluster_name}-lb"
 export GOVC_NETWORK="${LEASED_RESOURCE}"
-govc vm.clone -on=false -vm="${vm_template}" ${LB_VMNAME}
+if [ ${LEASE_NUMBER} -ge 151 ] && [ ${LEASE_NUMBER} -le 157 ]; then
```

Comment:
Can we add some comment lines to indicated what is the `LEASE_NUMBER` checking for?

Reply:
ok